packages/design-system-react-native/src/components/BadgeNetwork/BadgeNetwork.tsx

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable @typescript-eslint/prefer-nullish-coalescing */
+import React from 'react';
+
+import AvatarNetwork, { AvatarNetworkSize } from '../AvatarNetwork';
+import type { BadgeNetworkProps } from './BadgeNetwork.types';
+
+const BadgeNetwork = (props: BadgeNetworkProps) => (
```

Comment:
Suggestion: I think we want to be intentional about using some of the `AvatarNetwork` props here such as `name` and possibly `fallbackText` so it might be good to expose them